bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/BootstrapAjaxButton.html

```diff
@@ -1,10 +1,5 @@
 <html xmlns:wicket="http://wicket.apache.org">
-    <head>
-        <title>TypedPageButton</title>
-    </head>
-    <body>
-        <wicket:panel>
-            <i wicket:id="icon"></i><span wicket:id="splitter">&nbsp;</span><span wicket:id="label"></span>
-        </wicket:panel>
-    </body>
+<wicket:panel>
```

Comment:
IMHO changing this brings no value and if you want to port back this to 3.x branch than you will have conflicts.

Reply:
I will revert, NP
I thought setting page title in this fragment is useless